backend/src/main/kotlin/org/loculus/backend/api/SubmissionTypes.kt

```diff
@@ -291,10 +283,42 @@ enum class Status {
     }
 }
 
+enum class ProcessingResult {
+    /**
+     * The sequence has no warnings or errors
+     */
```

Comment:
```suggestion
    // The sequence has no warnings or errors
```

Are these docstrings auto-generated or have special meaning or why do we use 2 extra lines here 😄

Reply:
I'm not at all against comments (we currently have very few because of Fabian's preference), but adding extra white space for no reason is maybe overdoing it 🙃

Reply:
Ok, so ChatGPT told me these are KDoc comments, which are used by automatic tools, so that's a reason not to use `//` - however it seems possible to use `/** The sequence has no warnings or errors. */` instead here

Reply:
[781edae](https://github.com/loculus-project/loculus/pull/3125/commits/781edae3a64c90385a3ae01ed85471bae68a4e30)